tool-openssl/crl_test.cc

```diff
@@ -0,0 +1,181 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include "openssl/x509.h"
+#include <gtest/gtest.h>
+#include <openssl/pem.h>
+#include "internal.h"
+#include "test_util.h"
+#include "../crypto/test/test_util.h"
+#include <cctype>
+
+static X509_CRL* createTestCRL() {
+  X509_CRL *crl = X509_CRL_new();
```

Comment:
You can also use a smart pointers for this:
```
bssl::UniquePtr<X509_CRL> crl(X509_CRL_new());
```
But since this value is returned, it'll need to be released:
```
return crl.release();
```